<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Panel</class>
 <widget class="QWidget" name="Panel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>561</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/PassengerPanel/PassengerPanel.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ChargeBtn">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>50</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Charge Account</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RateBtn">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>140</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Rate Last Trip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CodeBtn">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>210</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Discount Code</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RequestBtn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>350</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Request Trip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelTrip">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>400</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel Trip</string>
   </property>
  </widget>
  <widget class="QLabel" name="UserLable">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="DateLable">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="CreditLable">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QLabel" name="TripStatus">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLable</string>
   </property>
  </widget>
  <widget class="QPushButton" name="UpdateStatusBtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Status</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
